cnf/galera_node.cnf

```diff
@@ -0,0 +1,132 @@
+[mysqld]
```

Comment:
I guess .cnf files are going from the place defined in the configuration (for every machine separately). I guess they should be removed from mdbci